docs/_sass/color_schemes/geoblazor.scss

```diff
@@ -84,7 +85,43 @@ $body-background-color: $white;
             
 $search-background-color: $white;
 $table-background-color: $white;
-$code-background-color: $grey-lt-000;
+//$code-background-color: $grey-dk-300;
```

Comment:
Should probably remove commented-out lines. If there's a reason you left it, then add a comment to specify the purpose.